src/Cli.Tests/UpdateEntityTests.cs

```diff
@@ -1771,6 +1771,7 @@ public void TestUpdateEntityPermissionWithWildcardAndOtherCRUDAction()
         [DataRow("stored-procedure", new string[] { "param1:value1,param2:223" }, null, "anonymous", "create,read", DisplayName = "Stored procedure with more than 1 CRUD operation")]
         [DataRow("stored-procedure", new string[] { "param1:value1,param2:223" }, null, "anonymous", "*", DisplayName = "Stored procedure with wildcard CRUD operation")]
         [DataRow("view", new string[] { "param1:value1" }, null, "anonymous", "*", DisplayName = "Source Parameters incorrectly used with View")]
+        [DataRow("view", null, null, "anonymous", "*", DisplayName = "Mandatory KeyFields for views not provided.")]
```

Comment:
I see you have a test case for when sourcetype isn't defined, can you also add a test to confirm failure where parameters and keyfields are included for a view?